processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceReference.java

```diff
@@ -108,8 +108,14 @@ public SourceReference build() {
             boolean isValid = true;
             boolean foundEntryMatch;
 
+            String sourceNameTrimmed = sourceName.trim();
+            if ( !sourceName.equals( sourceNameTrimmed ) ) {
+                messager.printMessage( method.getExecutable(), mapping.getMirror(), mapping.getSourceAnnotationValue(),
+                                       Message.PROPERTYMAPPING_WHITESPACE_TRIMMED, mapping.getSourceName()
```

Comment:
I would pass the `sourceName` (`targetName` in the `TargetReference`). I know that it is the same, but it reads better I think.